signalfx-org-metrics/README.md.jinja

```diff
@@ -67,21 +67,21 @@ These metrics track limits that Infrastructure Monitoring enforces for your orga
 * `sf.org.limit.hosts` (gauge): Maximum number of hosts that can send data to your organization.
```

Comment:
nit:  might bear repeating which limit "this" refers to :)

Reply:
Thanks for the reminder. Not gonna change it this time, but I'll pay more attention to my "this"'s in the future.